.github/workflows/python-poetry-release.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
 
       - name: Create changelog
         id: build-changelog
-        uses: bakdata/ci-templates/actions/changelog-generate@1.35.0
+        uses: bakdata/ci-templates/actions/changelog-generate@feat/changelog-eof
```

Comment:
Todo

Reply:
thanks... 
will do it if you think everything else is ok.

Reply:
done